.taprc

```diff
@@ -0,0 +1,15 @@
+esm: false
+ts: false
+jsx: false
+check-coverage: true
+100: true
+reporter: classic
+nyc-arg:
+- --all=true
+output-dir: .tap
```

Comment:
I would use `output-file: .tap` here. It prevents printing of the tap output when multiple files are present

Reply:
ah yes, I remember that issue

Reply:
Fixed here, in Jenkins file, and in .gitignore